source/fundamentals/indexes.txt

```diff
@@ -371,3 +384,97 @@ The following example creates a unique, descending index on the ``theaterId`` fi
 
 Refer to the :manual:`Unique Indexes page </core/index-unique>` in the MongoDB server manual for more information.
 .. driver-content-end
+
+Remove an Index
+---------------
+
+You can remove any unused index that isn't a :ref:`unique index
+<unique-indexes>`. 
+
+The following sections show the three ways to remove indexes:
+
+- Using an index specification document
+- Using an indexed name field
+- Removing all indexes
+
+Index Specification Document
```

Comment:
In the style guide, there's mention of not using headings and subheadings to create sentences ("Remove an Index [using an] Index Specification Document") because it should be clear what the subject is for every heading. Instead consider:
```suggestion
Remove an Index Using an Index Specification Document
```

Applies to all headings.